lib/encoding/pem.js

```diff
@@ -26,6 +26,9 @@ const assert = require('bsert');
 const base64 = require('./base64');
 const lines = require('./lines');
 
+const EOL = require('os').EOL;
```

Comment:
Really curious why those tests were failing. As far as I know we explicitly use LF everywhere, I wonder where CRLF is getting inserted.

Reply:
It appears to be an issue with travis windows configuration: https://travis-ci.community/t/files-in-checkout-have-eol-changed-from-lf-to-crlf/349

Reply:
Maybe we could just modify the tests to do `.replace(/\r\n/g, '\n')` on the PEM text vectors before comparing.

Reply:
Yeah worst case I'll do that but looks like people have found some workarounds. Trying to see if it works for us.